### PR TITLE
Vinai/constraints table

### PR DESCRIPTION
**Pull request by VinaiRachakonda**
Created new pr due to past problems

### CODE REVIEW

**Comment by VinaiRachakonda** [comment]
For reference : https://dev.mysql.com/doc/refman/8.0/en/information-schema-table-constraints-table.html

File: enginetest/queries.go [hunk 1]
```diff
@@ -5407,6 +5429,44 @@ var InfoSchemaScripts = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "Create a table with a check and validate that it appears in check_constraints and table_constraints",
+		SetUpScript: []string{
+			"CREATE TABLE mytable (pk int primary key, test_score int, height int, CONSTRAINT mycheck CHECK (test_score >= 50), CONSTRAINT hcheck CHECK (height < 10))",
```
Comment (zachmu) [inline enginetest/queries.go:5435]: You should have a third check that isn't selected by your first test query

File: enginetest/queries.go [hunk 2]
```diff
@@ -5407,6 +5429,44 @@ var InfoSchemaScripts = []ScriptTest{
...
+				Expected: []sql.Row{
+					{"def", "mydb", "hcheck", "mydb", "mytable", "CHECK", "YES"},
+					{"def", "mydb", "mycheck", "mydb", "mytable", "CHECK", "YES"},
+					{"def", "mydb", "PRIMARY", "mydb", "mytable", "PRIMARY KEY", "YES"},
+				},
+			},
+		},
+	},
+	{
+		Name: "information_schema.table_constraints ignores multi-index",
```
Comment (zachmu) [inline enginetest/queries.go:5456]: bad name, "ignores non-unique indexes"

**Review by zachmu — APPROVED** [review]
LGTM

**Pull request merged**